modules/reissue-internal-certificates-secured-clusters-automatic-upgrades.adoc

```diff
@@ -1,29 +1,36 @@
 // Module included in the following assemblies:
 //
 // * configuration/reissue-internal-certificates.adoc
+
 :_mod-docs-content-type: PROCEDURE
 [id="reissue-internal-certificates-secured-clusters-automatic-upgrade_{context}"]
 = Reissuing internal certificates for secured clusters by using automatic upgrades
 
-You can reissue internal certificates for Sensor, Collector, and Admission controller by using automatic upgrades.
+Secured clusters contain the Collector, Sensor, and Admission Control components.
+You can reissue internal certificates for these components by using automatic upgrades.
 
-[NOTE]
+[IMPORTANT]
 ====
-Automatic upgrades are only applicable to static manifest-based deployments using the `roxctl` CLI. See "Installing Central" in the "Installing by using the roxctl CLI" section of the _Installing_ chapter.
+* Automatic upgrades are only applicable to static manifest-based deployments by using the `roxctl` CLI. 
++
+For more information, see "Install Central using the roxctl CLI".
+
+* If you regularly upgrade {rh-rhacs-first}, your certificates are reissued in {product-title-short} 4.7.0. All pods pick up these certificates when you upgrade to {product-title-short} 4.7.1, unless the pods have already been restarted for other reasons. This issue mainly affects you if your certificates expire between these versions. 
++
+After upgrading to {product-title-short} 4.7.1, all pods use the new certificates, which are valid for another year. After the legacy init bundles are no longer supported, you can perform a hot reload of the certificates.
```

Comment:
This does not belong in this section. I will also create a KCS with this problem, my proposed text is here: https://docs.google.com/document/d/1Tt_ZVI6Xr_sTKK3Sl2dhvfVk29gwbJTa3Mlno8tBd_w/edit?usp=sharing